### PR TITLE
[DOCFIX] Add doc on running sudo without password

### PR DESCRIPTION
**Pull request by xwzbupt**
### What changes are proposed in this pull request?

add a statement describing that user must use sudo command without inputting password.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If not, the other nodes will waiting for inputting password but nobody will do it.

### Does this PR introduce any user facing changes?

   no

### CODE REVIEW

**Comment by xwzbupt** [comment]
alluxio-bot, check it please.

File: docs/en/deploy/Running-Alluxio-On-a-Cluster.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ To deploy Alluxio in production, we highly recommend running Alluxio masters in
   For basic functionality, make sure RPC port (default:19998) is open on all nodes.
 * Allow `sudo` privilege for the OS user that Alluxio will be running as.
   This is only needed if you expect Alluxio to mount a RAMFS on the workers automatically.
+* Enable sudo command can be used without inputting password.
```
Comment (dbw9580) [inline docs/en/deploy/Running-Alluxio-On-a-Cluster.md:39]:
```suggestion
* Ensure that the user Alluxio will be running as can execute `sudo` without inputting a password.
```

Also can you please add a link to the section from the other doc where there are instructions on how to set `NOPASSWD` config for `sudo`?

**Review by dbw9580 — CHANGES_REQUESTED** [review]

**Comment by dbw9580** [comment]
Please prefix the title with `[DOCFIX]`. This skips all CI tests.

**Comment by xwzbupt** [comment]
alluxio-bot, check it please.

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**